src/pc_screen_effect.c

```diff
@@ -9,10 +9,10 @@
 #define tState data[0]
 #define tXSpeed data[1]
 #define tYSpeed data[2]
-#define tWin0Left data[3]
-#define tWin0Right data[4]
-#define tWin0Top data[5]
-#define tWin0Bottom data[6]
+#define tWinLeft data[3]
+#define tWinRight data[4]
+#define tWinTop data[5]
+#define tWinBottom data[6]
```

Comment:
Any reason for this change?

Reply:
Is there any reason to keep 0 in between these names??
Like there is no tWin1Left, tWin2Left or something
And of course it's consistent with pokeemerald